### PR TITLE
chore(docs): update readme and api docs for merged repos

### LINKED ISSUE #73: Readme files and documentation are out of date
Now that the three old database repos have been merged into one, the readme files and documentation are wrong.

### PR DESCRIPTION
**Pull request by philbooth**
Fixes #73.

Not sure how much use the diff is here, you might find it easier to review by just reading the docs themselves.

### CODE REVIEW

**Comment by rfk** [comment]
Thanks @philbooth.  Are some of these paragraphs trimmed to a really narrow column count?  For example, this appears to be split over three lines:

```
+This is the backend store
+that is loaded by the default export
+from the npm package,
```

**Comment by philbooth** [comment]
>  Are some of these paragraphs trimmed to a really narrow column count?

Yes, it was purely force-of-habit sorry. Do you want me to concat them into longer lines?

Fwiw, the thinking is to make diffs of text documents easier by splitting lines into logical chunks that can change independently. [Apparently it was a Kernighan thing originally](http://rhodesmill.org/brandon/2012/one-sentence-per-line/). :smile:

Anyway, happy to change them however, let me know.

**Comment by philbooth** [comment]
I've also re-pushed with the list of prerequisites reinstated. I realised it had got lost when I saw Vlad's PR on `fxa-auth-db-server`.

**Comment by rfk** [comment]
I've never seen that before, but I kinda like it.  It almost reads like free-verse poetry... :-)

File: README.md [hunk 1]
```diff
@@ -1,88 +1,204 @@
...
 
 ```sh
-node bin/db_patcher.js
+npm start
 ```
 
-This should create the database (if it doesn't yet exist), and apply each patch located
-in `db/schema/*.sql` in the correct order. If this command fails and can't connect to the
-database, please check your mysql configuration and connectivity on the command line.
+This will set-up the database for you
```
Comment (rfk) [inline README.md]: nit: "set-up" should be two words, not hyphenated

**Comment by rfk** [comment]
otherwise LGTM

**Pull request merged**